en/docs/Administer/ProductAdministration/UpgradingFromAPreviousRelease/upgrading-from-the-previous-release.md

```diff
@@ -1679,7 +1679,7 @@ From WSO2 API Manager 2.0.0 onwards, statistics can be configured only for RDBM
             sh wso2server.sh
     ```
 
-Follow the instructions below to upgrade your WSO2 API Manager server **from APIM 1.8.0/1.9.0/1.9.1** **to 2.6.0** .
+Follow the instructions below to upgrade your WSO2 API Manager server **from APIM 1.8.0/1.9.0/1.9.1to 2.6.0** .
```

Comment:
space is missing

Reply:
done

Reply:
Fixed in Commit on Sep 9, 2019 - https://github.com/wso2/docs-apim/pull/19/commits/d2c3ff8450dd3cdcef80a86027b482e4ec4f5410